archive/k/kotlin/fibonacci.kt

```diff
@@ -0,0 +1,25 @@
+import kotlin.system.exitProcess
+
+// First arg is number of iterations to run
+fun main(args: Array<String>) {
+    if (args.size != 1 && args[0].toIntOrNull() == null)
```

Comment:
This logic is bogus. The program only exits if there's 0 or more than 1 arguments **and** the first argument isn't an integer.

Reply:
Oh yeah this should be an OR not an AND, my bad.

Reply:
Hey @chrboe! Thanks for helping out here. Can we be more welcoming or tactful with our comments? That first sentence seems a little aggressive.